src/test/java/qna/domain/AnswerTestFixture.java

```diff
@@ -0,0 +1,6 @@
+package qna.domain;
```

Comment:
```suggestion
package qna.fixture;
```
여러 테스트에서 사용되고 있는 픽스처들인 것 같은데요, 픽스처도 패키지를 따로 가져가보면 어떨까요?

Reply:
테스트가 더 많아지는 걸 고려해서 분리하면 좋겠네요!